CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+**Table of Contents**
+<!-- TOC depthFrom:2 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Versioning](#versioning)
+- [Branches](#branches)
+- [Pull Requests](#pull-requests)
+	- [Working on a Pull Request](#working-on-a-pull-request)
+	- [Reviewing a Pull Request](#reviewing-a-pull-request)
+	- [Merging a Pull Request](#merging-a-pull-request)
+- [Zenhub Issue Board](#zenhub-issue-board)
+	- [Pipelines](#pipelines)
+- [Reporting A Bug](#reporting-a-bug)
+	- [Before Submitting A Bug Report](#before-submitting-a-bug-report)
+	- [How Do I Submit A Bug Report?](#how-do-i-submit-a-bug-report)
+
+<!-- /TOC -->
+## Versioning
+
+This project uses [Semantic Versioning](http://semver.org/) or _SemVer_ for short. If you do not know what SemVer is, a detailed explanation is available on their website.
+Any new releases must adhere to this versioning scheme.
+
+## Branches
+Our project currently consists of a _master_ and several _feature_ branches. 
+The master branch always contains a stable version of the project. Only commits with small, insignificant changes should be done directly on the master branch. 
```

Comment:
If we want to handle this this way. We have to have some sort of development branch. however as I said in the last meeting I think this is uneccessary because a stable version will just receieve a tag.

Reply:
A dev branch is better. You can just clone the project and the master is stable and you don't have to search for the last tag.